### PR TITLE
Add c vlink/ Navbar fixes

### LINKED ISSUE #4: Update route to match nav as you scroll down the page.
Clicking "Work" takes you to `#projects` but just scrolling down doesn't update the url. This makes copy-paste sharing harder.

### PR DESCRIPTION
**Pull request by andrayantelo**
Resolves #4 
- added link in navbar for CV
- the links in the navbar become active as you scroll
- also when you click on a link in the navbar, the active class gets added to the link that was clicked

### CODE REVIEW

File: favicon.ico [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (johntyree) [inline favicon.ico:1]: why?

File: script.js [hunk 2]
```diff
@@ -1 +1,77 @@
...
+        // also add link to history when it is clicked on
+        if (history.pushState) {
+            history.pushState({url: target}, null, target);
+        }
+        
+        $('html, body').stop().animate({
+            'scrollTop': $target.offset().top
+        }, 500, 'swing', function () {
+            window.location.hash = target;
+            $(document).on("scroll", onScroll);
```
Comment (johntyree) [inline script.js:29]: Why do you attach this again here? You have a handler for `scroll` events on line 7

**Review by johntyree — CHANGES_REQUESTED** [review]

File: script.js (hunk 2 above)
Comment (andrayantelo) [reply script.js:29]: it gets detached on line 8

File: favicon.ico (hunk 1 above)
Comment (andrayantelo) [reply favicon.ico:1]: because I was getting an error. Isn't this what we did last time?

**Pull request merged**